sphinxarg/parser.py

```diff
@@ -37,7 +37,7 @@ def _try_add_parser_attribute(data, parser, attribname):
     if not isinstance(attribval, str):
         return
     if len(attribval) > 0:
-        data[attribname] = attribval
+        data[attribname] = attribval % {'prog': data['prog']}
```

Comment:
```suggestion
        try:
            data[attribname] = attribval % {'prog': parser.prog}
        except AttributeError:
            data[attribname] = attribval
```

Reply:
"prog" is not included in the `subaction` dict passed into `parse_parser` on line 95.

Reply:
Some testing indicates that your suggested change isn't enough but thank you for catching this!